WTWSMS/events/WtWSMS_migration_events.txt

```diff
@@ -664,15 +664,23 @@ province_event = {
 					set_character_flag = decided_culture
 				}
 				
-				usurp_title_plus_barony_if_unlanded = PREV
+				ROOT = { county = { usurp_title_plus_barony_if_unlanded = PREVPREV } }
 				set_defacto_liege = THIS
 				reverse_culture = ROOT
 				reverse_religion = ROOT
 				
+				#spawn_unit = { #original code by Zusk, doesn't work properly
+				#	province = PREV
+				#	owner = ROOT
+				#	match_character = ROOT
+				#	match_mult = 0.75
+				#	attrition = 0.000
+				#}
+				# I suggest rewriting the entire unit block
```

Comment:
Ok, did you have a concrete idea in mind?

Reply:
Nothing concrete, I just think we should do it because we don't know for sure who should be the match_character and a more complex block wouldn't hurt. But the fixed block below is fully functional for now 👌.